javaagent-bootstrap/src/main/java/io/opentelemetry/javaagent/bootstrap/AgentClassLoader.java

```diff
@@ -94,7 +94,7 @@ public AgentClassLoader(File javaagentFile, String internalJarFileName, ClassLoa
       jarBase =
           new URL("x-internal-jar", null, 0, "/", new AgentClassLoaderUrlStreamHandler(jarFile));
       codeSource = new CodeSource(javaagentFile.toURI().toURL(), (Certificate[]) null);
-      manifest = getManifest(jarFile, jarEntryPrefix + META_INF_MANIFEST_MF);
```

Comment:
Can `META_INF_MANIFEST_MF` constant be removed too?

Reply:
I'm surprised errorprone didn't complain about this 😁

Reply:
removed